pkg/compare/testdata/CheckNoOverwrite/reference/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+# cluster-compare: allow-undefined-extras=true
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-storage
+  annotations:
+    workload.openshift.io/allowed: management
+  labels:
+    openshift.io/cluster-monitoring: "true"
```

Comment:
can you add a comment here that this is the filed that is checked to not be overwrite